.github/workflows/lint.yml

```diff
@@ -50,46 +50,3 @@ jobs:
         npm_config_yes=true npx github:10up/eslint-json-to-md --path ./eslint-report.json --output ./eslint-report.md
         cat eslint-report.md >> $GITHUB_STEP_SUMMARY
       if: ${{ failure() }}
-  phpcs:
-    name: phpcs
-    runs-on: ubuntu-latest
-
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v3
-
-    - name: Set standard 10up cache directories
-      run: |
-        composer config -g cache-dir "${{ env.COMPOSER_CACHE }}"
-
-    - name: Prepare composer cache
-      uses: actions/cache@v2
-      with:
-        path: ${{ env.COMPOSER_CACHE }}
-        key: composer-${{ env.COMPOSER_VERSION }}-${{ hashFiles('**/composer.lock') }}
-        restore-keys: |
-          composer-${{ env.COMPOSER_VERSION }}-
-
-    - name: Set PHP version
-      uses: shivammathur/setup-php@v2
-      with:
-        php-version: '7.4'
-        coverage: none
-        tools: composer:v2, cs2pr
-
-    - name: Install dependencies
-      run: composer install --ignore-platform-reqs
-
-    - name: Check PHPCS standard
-      run: ./vendor/bin/phpcs -i
-
-    - name: Check Compatibility of main plugin file.
-      run: ./vendor/bin/phpcs ./distributor.php --runtime-set testVersion 5.6- --standard=PHPCompatibilityWP
-
-    - name: PHPCS check
-      id: phpcs-sniffs
-      run: ./vendor/bin/phpcs . --report-full --report-checkstyle=./.github/phpcs-report.xml
-
-    - name: Show PHPCS results in PR
-      if: ${{ always() }}
-      run: cs2pr ./.github/phpcs-report.xml
```

Comment:
Are you able to retain this step -- the setup-php action will need to include cs2pr.